docs/api/wallet-rpc.yaml

```diff
@@ -343,6 +343,61 @@ paths:
           $ref: '#/components/responses/409-NoConfig'
         '503':
           $ref: '#/components/responses/503-ServiceUnavailable'
+  /wallet/{walletname}/taker/schedule:
+    post:
+      security:
+        - bearerAuth: []
+      summary: create and run a schedule of transactions
```

Comment:
As per comment to the code, it's strange that we are describing and naming this after a procedure to pass in a schedule and run it, rather than what it actually currently is - pass in destinations (and perhaps other options in future), and let the code construct the schedule according to the tumbler algorithm.